Draft: Update Rhythmbox3 runtime to 46

yakushabb opened the pull request:
Update Rhythmbox3 runtime to 46 because runtime 45 will be EOL on 2024-09-14.

hadess commented on appdata-fixes.patch:
```diff
@@ -1,81 +1,88 @@
+From 9476009d62145437f4fe3531fa17c7ffebc3d8da Mon Sep 17 00:00:00 2001
+From: Sabri Ünal <yakushabb@gmail.com>
+Date: Thu, 25 Jul 2024 23:32:25 +0300
+Subject: [PATCH] Fix appdata papercuts
```
If those are all already upstream, could you please use commit `77954dc2b536170afb84d67c03d92dafefd3fc0d` as the basis to create a patch set, as was already done for those few fixes I submitted?

hadess commented on org.gnome.Rhythmbox3.json:
```diff
@@ -33,7 +33,17 @@
         "cxxflags": "-O2 -g"
     },
-    "cleanup": [ "/include", "/share/bash-completion" ],
+    "cleanup": [
```
Please remove "Testing" from the commit message.

hadess commented:
Unless you can justify it, please remove the [Update submodules](https://github.com/flathub/org.gnome.Rhythmbox3/pull/71/commits/41c4f0d35027dfe2ed2451a23f0d840c6db7f910) commit, the only shared module we used wasn't updated. If you can justify it, please call it "shared-modules".

Commit:
Update Rhythmbox3 runtime to 46

Update Rhythmbox3 runtime to 46 because runtime 45 will be EOL on 2024-09-14.

Commit:
Reduce package size

Remove some redundant files to reduce package size.

Commit:
Use another commit to exclude patches

Use the commit `77954dc2b536170afb84d67c03d92dafefd3fc0d`
to avoid patching the appdata file.
